temba/api/v2/views.py

```diff
@@ -3573,3 +3581,454 @@ def post(self, request, *args, **kwargs):
+                "items": [
+                    {
+                        "name": "Gendalf the White",
+                        "age": 500
+                    }
+                ]
+            }
+
+
+        Response status of success creation would be `201`:
+
+            [
+                {
+                    "success": {
+                        "objectId": "MVPuxmVfG7",
+                        "createdAt": "2020-12-10T20:14:45.999Z"
+                    }
+                }
+            ]
+
+        ## Lookup Record Updating
+
+        A `PUT` update existing row in collection:
+
+         * **collection_name** - the name of collection
+         * **objectId** - identifier of row that is going to be updated
+         * **item** - a data to update existing row
+
+        Replace data of existing row for specific collection:
+
+            PUT /api/v2/database_records.json
+            {
+                "collection_name": "Lookups List",
+                "objectId": "J7vDtb5Aek",
+                "item": {
+                    "name": "Gendalf the Gray"
+                }
+            }
+
+
+        Response status of success updating would be `202`:
+
+            {
+                "updatedAt": "2020-12-10T20:24:32.595Z"
+            }
+
+        ## Lookup Record Deleting
+
+        A `DELETE` deletes collection:
+
+         * **collection_name** - the name of collection
+         * **objectId** - identifier of record to delete
+
+        Create new collection for current org:
+
+            DELETE /api/v2/database_records.json
+            {
+                "collection_name": "Lookups List",
+                "objectId": "J7vDtb5Aek"
+            }
+
+        Response status of success deletion would be `204`
+    """
+
+    permission = "orgs.org_lookups"
+
+    def list(self, request, *args, **kwargs):
+        org, collection_name, collections_list, error_response = self.get_default_params(config="collection_exist")
+        if error_response:
+            return error_response
+
+        collection = self.get_collection_full_name(org=org, collection=collection_name)
+        parse_headers = {
+            "X-Parse-Application-Id": settings.PARSE_APP_ID,
+            "X-Parse-Master-Key": settings.PARSE_MASTER_KEY,
+            "Content-Type": "application/json",
+        }
+        results_url = f"{settings.PARSE_URL}/classes/{collection}?order=order&limit=1000"
+        response = requests.get(results_url, headers=parse_headers)
+        result = response.json()
+
+        return Response(result, status=status.HTTP_200_OK)
+
+    def post(self, request, *args, **kwargs):
+        org, collection_name, collections_list, error_response = self.get_default_params(config="collection_exist")
+        if error_response:
+            return error_response
+
+        items_to_push = self.request.data.get("items", [])
+        if not items_to_push:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Let's add a message to this response, this way the user would know what's wrong.

Reply:
Added response details.